File: armstrong.py
#Check Armstrong number of n digits
number = 1634
order len(str(num))
sum = 0
temp = number
while temp>0:
  digit = temp%10
  sum += digit**order
  temp //= 10
if number == sum:
  print(number,"is  an Armstrong number")
else:
  print(number,"is not an Armstrong number")

File: armstrong.c
#include <stdio.h>
int main() {
    int num, originalNum, remainder, result = 0;
    printf("Enter a three-digit integer: ");
    scanf("%d", &num);
    originalNum = num;

    while (originalNum != 0) {
        remainder = originalNum % 10;
        
       result += remainder * remainder * remainder;
        
       originalNum /= 10;
    }

    if (result == num)
        printf("%d is an Armstrong number.", num);
    else
        printf("%d is not an Armstrong number.", num);

    return 0;
}
